static/app/views/organizationContext.tsx

```diff
@@ -37,6 +38,17 @@ interface Props {
   children: React.ReactNode;
 }
 
+const proxyHandler: ProxyHandler<Organization> = {
+  get(organization, prop) {
+    // XXX(epurkhiser): Record a sentry message when we've accessed the
+    // organiztion. Once we stop getting these we can remove the
+    // LegacyOrganizationContextProvider
+    captureMessage('Legacy organization accessed!');
```

Comment:
could also include the `prop` to try and narrow down any callsites. Seeing `slug` in there won't be helpful, but if the message is triggered with something exotic then it could be the silver bullet.

Reply:
It will already include the stack trace I believe